public/app/plugins/datasource/loki/syntax.ts

```diff
@@ -49,7 +49,67 @@ const AGGREGATION_OPERATORS: CompletionItem[] = [
   },
 ];
 
+export const PIPE_PARSERS: CompletionItem[] = [
+  {
+    label: 'json',
+    insertText: 'json',
+    documentation: 'Extracting labels from the log line using json parser. Only available in Loki 2.0+.',
+  },
+  {
+    label: 'regexp',
+    insertText: 'regexp',
+    documentation: 'Extracting labels from the log line using regexp parser. Only available in Loki 2.0+.',
```

Comment:
Could we automatically move the cursor into the regex string. e.g `| regexp "here"` ?

Reply:
Yes, good idea. Added in https://github.com/grafana/grafana/pull/27884/commits/796afda6df34f9ef23a23bef5c6b67d582ee8d90